source/server/server_configuration_parser.cpp

```diff
@@ -164,12 +164,12 @@ FeatureToggles::CodeReadiness ServerConfigurationParser::parse_code_readiness()
       using ReadinessMap = std::unordered_map<std::string, FeatureToggles::CodeReadiness>;
       const auto READINESS_MAP = ReadinessMap{
           {"release", FeatureToggles::CodeReadiness::kRelease},
-          {"restrictedrelease", FeatureToggles::CodeReadiness::kRestrictedRelease},
-          {"restricted_release", FeatureToggles::CodeReadiness::kRestrictedRelease},
+          {"restrictedrelease", FeatureToggles::CodeReadiness::kRelease},
```

Comment:
I think separating the restricted release stuff out of the CodeReadiness concept is correct. But for the config files, should we still accept the "restricted" strings to support already created config files?

Reply:
I'm going to say yes for now. We can revisit this later if it's deemed confusing and we don't care about supporting anyone who happened to really on "restricted.." for code_readiness in their config files.

Reply:
Just had a thought on this - If you want to go above and beyond, you could print a warning message when a configuration file is using these indicating they're deprecated. I don't know how much effort that would be so feel free to just take this approach.